x-pack/plugins/triggers_actions_ui/public/application/lib/action_variables.ts

```diff
@@ -156,6 +157,13 @@ function getAlwaysProvidedActionVariables(): ActionVariable[] {
     ),
   });
 
+  result.push({
+    name: AlertProvidedActionVariables.alertFlapping,
+    description: i18n.translate('xpack.triggersActionsUI.actionVariables.alertFlappingLabel', {
+      defaultMessage: 'A flag on the alert that indicates whether the alert is flapping.',
```

Comment:
It doesn't seem like we currently define "flapping" anywhere in the docs. Does it mean the state of the alert is changing repeatedly? I think it would be helpful to clarify that term in the docs and in the message.

Reply:
Ah, good point!  Yes, flapping is when an alert is going active, then recovering, then active, then recovering, in "quick succession".  I'm not sure we have all the bits nailed down right now in terms of UX, and more importantly defaults:how long we look to past executions for "flaps", how many of those "flaps" it takes for an alert to be considered "flapping", and how an alert leaves "flapping" state.  

There's a link to the RFC at the top of https://github.com/elastic/kibana/issues/142876 that describes this at a somewhat high level.  We'll need this boiled down to something understandable for the docs, by the time we fully ship the flapping bits, which won't be until at least 8.7.  I'm guessing we will need a paragraph or two :-). Until then, there will be some bits of flapping leaking out as we work on it, like the messaging above indicates.

Until then, I think just making sure the existing text messages are up to standards is probably the best we can do.